### PR TITLE
add folder for github-and-markdown

### PR DESCRIPTION
**Pull request by AHen-C0de**
contains an Info.md with main information about github and markdown and a cheatsheet for markdown (yet to be completed...)

### CODE REVIEW

**Review by Sukhpreet92 — APPROVED** [review]

File: github-and-markdown/Info.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+| [x] done                        | `[x] checkbox`                    |
+| **bold text**                   | `**bold text**`                   |
+| _italicized text_               | `_italicized text_`               |
+| [link](https://www.example.com) | `[link](https://www.example.com)` |
+| image                           | `![link](url to image)`           |
+| block quote                     | `> block quote`                   |
+| divider                         | `---`                             |
+| `inline code block`             | `` `inline code block` ``         |
+| `code block`                    | ` ``` code block ``` `            |
+
```
Comment (Ozanyurdakul98) [inline github-and-markdown/Info.md:27]: I like this table a lot!!!

File: github-and-markdown/Markdown-Cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### _Further_:
+
+---
+
+...
```
Comment (Ozanyurdakul98) [inline github-and-markdown/Markdown-Cheatsheet.md:7]: I think this is in the wrong place!

**Review by Ozanyurdakul98 — APPROVED** [review]
It looks really good !!

File: github-and-markdown/Markdown-Cheatsheet.md (hunk 2 above)
Comment (AHen-C0de) [reply github-and-markdown/Markdown-Cheatsheet.md:7]: But this is the markdown cheat sheet. I would keep it in the session for "github-and-markdown", no?

**Pull request merged**